clang/test/Analysis/stream.c

```diff
@@ -498,3 +498,15 @@ void gh_93408_regression_ZeroSized(struct ZeroSized *buffer) {
   fread(buffer, 1, 1, f); // expected-warning {{Stream pointer might be NULL}} no-crash
   fclose(f);
 }
+
+extern FILE *stdout_like_ptr;
+void no_aliasing(void) {
```

Comment:
This test could have a more meaningful name.

Reply:
I think nobody will ever come back to this test. I don't think I'll elaborate the name. The content should be enough already.